Zendesk 1.0 Build Retry (#1248)
mulesoft/docs-connectors: zendesk/1.0/modules/ROOT/pages/zendesk-connector-reference.adoc

[%header%autowidth.spread]
|===
| Name | Type | Description | Default Value | Required
| Default Headers a| Array of <<DefaultHeader>> |  |  |
| Query Parameters a| Array of <<DefaultQueryParam>> |  |  |
| Connection Timeout a| Number |  The timeout for establishing connections to the remote service |  30 |
| Connection Timeout Unit a| Enumeration, one of:

** NANOSECONDS
** MICROSECONDS
** MILLISECONDS
** SECONDS
** MINUTES
** HOURS
** DAYS |  A time unit that qualifies the Connection Timeout |  SECONDS |
| Use Persistent Connections a| Boolean |  If `false`, each connection will be closed after the first request is completed. |  true |
| Max Connections a| Number |  The maximum number of outbound connections to keep open at the same time |  -1 |
| Connection Idle Timeout a| Number |  A timeout for how long a connection can remain idle before it is closed |  30 |
| Connection Idle Timeout Unit a| Enumeration, one of:

** NANOSECONDS
** MICROSECONDS
** MILLISECONDS
** SECONDS
** MINUTES
** HOURS
** DAYS |  A time unit that qualifies the connection Idle Timeout |  SECONDS |
| Proxy Config a| <<Proxy>> |  Reusable configuration element for outbound connections through a proxy |  |

> | Stream Response a| Boolean |  Whether or not to stream received responses, which means that processing continues as soon as all headers are parsed and the body is streamed as it is received. When enabled, the response must eventually be read since, depending on the configured buffer size, the response might not fit into memory and processing stops until space is available. |  false |

Reviewer: ```suggestion
| Stream Response a| Boolean |  Whether or not to stream received responses, which means that processing continues as soon as all headers are parsed and the body is streamed as it is received. When enabled, the response must eventually be read since, depending on the configured buffer size, the response could not fit into memory and processing stops until space is available. |  false |
```

Author: "Might" fits better here.

** MILLISECONDS
** SECONDS
** MINUTES
** HOURS
** DAYS |  A time unit that qualifies the Connection Timeout |  SECONDS |
| Use Persistent Connections a| Boolean |  If `false`, each connection will be closed after the first request is completed. |  true |
| Max Connections a| Number |  The maximum number of outbound connections to keep open at the same time |  -1 |
| Connection Idle Timeout a| Number |  A timeout for how long a connection can remain idle before it is closed |  30 |
| Connection Idle Timeout Unit a| Enumeration, one of:

** NANOSECONDS
** MICROSECONDS
** MILLISECONDS
** SECONDS
** MINUTES
** HOURS
** DAYS |  A time unit that qualifies the connection Idle Timeout |  SECONDS |
| Proxy Config a| <<Proxy>> |  Reusable configuration element for outbound connections through a proxy |  |
| Stream Response a| Boolean |  Whether or not to stream received responses, which means that processing continues as soon as all headers are parsed and the body is streamed as it is received. When enabled, the response must eventually be read since, depending on the configured buffer size, the response might not fit into memory and processing stops until space is available. |  false |

> | Response Buffer Size a| Number |  The space, in bytes, for the buffer where the HTTP response is stored. |  -1 |

Reviewer: ```suggestion
| Response Buffer Size a| Number |  The space, in bytes, for the buffer to store the HTTP response. |  -1 |
```
